README.md

```diff
@@ -676,6 +677,36 @@ $ where black
         - Working directory: `$ProjectFileDir$`
 	- Uncheck "Auto-save edited files to trigger the watcher"
 
+
+
+### Wing IDE
+
+1. Install `black`.
+
+```console
+$ pip install black
+```
+
+2. Make sure it runs from the command line, e.g.
+
+```console
+$ black --help
+```
+
+3. In Wing IDE, activate **OS Commands** panel  and define the command  **black** with e.g. the following values, to execute black on the currently selected file, using key binding **F1**:
```

Comment:
Thanks for doing this! I wanted to configure Black for use with Wing and had to track down how, and was going to write basically your patch.  Note that the Wing 7.0.0 manual refers to Black in its section on PEP8 formatting: https://wingware.com/doc/edit/pep8 )

Wording change: 'activate OS' -> 'activate the OS'.  

I suggest rewording the second half of the second as just: `define the command **black** to execute black on the currently selected file:`

Then change the key-binding line below to something like: - Key Binding: F1 <or enter a different key or sequence of keys>

Reply:
Thanks for the review.

I updated the descriptions with these suggestions, and also included a link to the wing documentation on pep8 formatting.